release-note.txt

```diff
@@ -0,0 +1,17 @@
+Release v0.6.1 release notes.
```

Comment:
OpenPAI v0.6.1 release notes.

BTW, isn’t it 0.6.0?

Reply:
yeah, we already used the version number 0.6.0 (although not announced)

Reply:
I renamed to release-notes.txt.
But not changed to .md file. 

Maybe raw txt is enough for us?